src/libsndfile-1.def

```diff
@@ -1,6 +1,4 @@
-; Auto-generated by cmake
-
-LIBRARY ${PROJECT_NAME}-${PROJECT_VERSION_MAJOR}.dll
+LIBRARY libsndfile-1.dll
```

Comment:
@erikd:

        ; Auto-generated by @GEN_TOOL@

`GEN_TOOL` for CMake is set to `cmake`. `autoconf` for Autotools is OK?